examples/pii-example/README.md

```diff
@@ -7,16 +7,16 @@
 2. Change link below to your personal VGS Collect library link. You can find it at VGS Collect page of [VGS Dashboard](https://dashboard.verygoodsecurity.com/)
 
 ```html
-<script src="https://js.verygoodvault.io/vgs-collect/1/<organization-id>.js"></script>
+<script src="https://js.verygoodvault.io/vgs-collect/2.0/vgs-collect.js"></script>
 ```
 
 3. Change `<vault-id>` to your vault id to initialize your Collect form
 
 ```javascript
-const form = VGSCollect.create('<vault-id>', function(state) {});
+const form = VGSCollect.create('<vault-id>', 'sandbox', function(state) {});
```

Comment:
the better way - show customers all possible choose 
`'sandbox'` -> `<'sandbox' || 'live'>`